conjur/init/init_controller.py

```diff
@@ -87,23 +120,22 @@ def get_account_info(self, conjurrc_data):
             # pylint: disable=broad-except,logging-fstring-interpolation
             except Exception as error:
                 # pylint: disable=line-too-long,logging-fstring-interpolation
-                logging.debug(f"Unable to fetch the account from the Conjur server. Reason: {error}")
+                logging.debug(f"Unable to fetch the account from the Conjur server. Reason: {error}\n")
                 # If there was a problem fetching the account from the server, we will request one
-                conjurrc_data.account = input("Enter your organization account name: ")
+                conjurrc_data.account = input("Enter your organization account name: ").strip()
 
                 if conjurrc_data.account is None or conjurrc_data.account == '':
                     # pylint: disable=raise-missing-from
-                    raise RuntimeError("Error: account is required")
+                    raise RuntimeError("Error: account is required\n")
 
     def write_certificate(self, fetched_certificate):
         """
         Method to write the certificate fetched from the Conjur endpoint on the user's machine
         """
-        url = urlparse(self.conjurrc_data.appliance_url)
-        if self.conjurrc_data.cert_file is None and url.scheme == "https":
+        # pylint: disable=line-too-long
+        if self.conjurrc_data.cert_file is None and self.conjurrc_data.appliance_url.startswith("https"):
```

Comment:
this change is unnecessary, the previous code was better.
Don't try to parse the url yourself, use the dedicated lib for that.

Reply:
reverting